index.html

```diff
@@ -3651,39 +3656,73 @@ <h3>Parsing content</h3>
         </ol>
       </li>
       <li>
-        If |ndef|'s |typeNameField| is `1` ([=well-known type record=]):
+        If |ndef|'s |typeNameField| is `1` ([=well-known type record=]), then
         <ol>
           <li>
-            Set |record| to the result of the algorithm below switching on
-            |ndef|'s |type:string|:
-            <dl>
-              <dt>"`T`" (`0x54`)</dt>
-              <ul>
-                <li>
-                  Running <a>parse an NDEF text record</a> on |ndef|.
-                </li>
-              </ul>
-              <dt>"`U`" (`0x55`)</dt>
-              <ul>
-                <li>
-                  Running <a>parse an NDEF URL record</a> on |ndef|.
-                </li>
-              </ul>
-              <dt>"`Sp`" (`0x53` `0x70`)</dt>
-              <ul>
-                <li>
-                  Running <a>parse an NDEF smart-poster record</a> on |ndef|.
-                </li>
-              </ul>
-            </dl>
+            If |ndef|'s |type:string| is <strong>"`T`" (`0x54`)</strong>,
+            set |record| to the result of running
+            <a>parse an NDEF text record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`U`" (`0x55`)</strong>,
+            set |record| to the result of running
+            <a>parse an NDEF URL record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`Sp`" (`0x53` `0x70`)</strong>,
+            set |record| to the result of running
+            <a>parse an NDEF smart-poster record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`s`" (`0x73`)</strong>
+            and if |context| is equal to `"smart-poster"`,
+            set |record| to the result of running
+            <a>parse a smart-poster size record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`t`" (`0x74`)</strong>
+            and if |context| is equal to `"smart-poster"`,
+            set |record| to the result of running
+            <a>parse a smart-poster type record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`act`" (`0x65` `0x63` `0x74`)
+            </strong> and if |context| is equal to `"smart-poster"`,
+            set |record| to the result of running
+            <a>parse a smart-poster action record</a> on |ndef|.
+          </li>
+          <li>
+            Otherwise, if running the <a>validate local type</a> steps on
```

Comment:
Shall we explicitly call out what happens if it's not a valid local type?

Reply:
Yes, fixed.